exchange/c2cx/c2cx.go

```diff
@@ -390,3 +390,8 @@ func (br BalanceSummary) Spendable() Balances {
 		Total: br.Balance.Total.Sub(br.Frozen.Total),
 	}
 }
+
+type Orders struct {
```

Comment:
Use this as the return value for `GetOrderByStatusPaged` ?

Reply:
Yes, I wrap the result into this struct, because **client.GetOrderByStatusPaged** returns separated params: **orders**, **pageCount**.